libs/blocks/footer/footer.css

```diff
@@ -333,18 +286,6 @@ footer .footer-region-button.inline-dialog-active svg.icon-chevron-down {
   margin-left: 6px;
 }
 
-.region-selector-text {
-  padding: 0 32px;
-}
-
-.region-selector {
-  padding: 0 32px 32px;
-}
-
-.region-selector .content {
-  column-count: 1;
-}
-
```

Comment:
@seanchoi-dev Just these should be removed. Nothing scoped inside of footer. 
This will make it much more clear that styles coming from `modal.css` are what controls modals.